src/components/LandingSection/LandingSection.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import SectionWrapper from '../../styles/elements/SectionWrapper';
+import HamburgerMenu from '../HamburgerMenu';
+import NavBar from '../NavBar';
+
+const LandingSectionWrapper = styled(SectionWrapper)`
+  align-items: center;
+`;
+
+const LandingHeader = styled.h1`
+  font-family: 'Alfa Slab One';
+  font-weight: 400;
```

Comment:
I recommend keeping weights (also fonts) in some object to avoid checking available weights.